### PR TITLE
Report view_promotion for bundle discount for 1st and 2nd offers in price calculator and from cart page

### PR DESCRIPTION
**Pull request by alebedev**
## Describe your changes

Report when we show bundle discount promotion to user

## Justify why they are needed

Part of bundle discount performance tracking

## Checklist before requesting a review

- [ ] I have performed a self-review of my code

### CODE REVIEW

**Comment by alebedev** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4379?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#4379** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4379?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#4378** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4378?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#4377** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4377?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#4376** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4376?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @alebedev and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: apps/store/src/services/Tracking/trackExperimentImpression.ts [hunk 1]
```diff
@@ -3,6 +3,7 @@ import { type Tracking } from '@/services/Tracking/Tracking'
 import { EXPERIMENT_COOKIE_NAME } from './experiment.constants'
 import { getExperimentVariant } from './experiment.helpers'
 
+// Website-driven experiments, as configured in experiment.json
```
Comment (guilhermespopolin) [inline apps/store/src/services/Tracking/trackExperimentImpression.ts:6]: 👍🏻

**Review by guilhermespopolin — APPROVED** [review]
Nice, nice

**Comment by alebedev** [comment]
### Merge activity

* **Jun 19, 5:59 AM EDT**: @alebedev started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4379).

**Pull request merged**